src/spyglass/lfp/lfp_merge.py

```diff
@@ -7,44 +7,24 @@
 from spyglass.common.common_nwbfile import AnalysisNwbfile
 from spyglass.lfp.v1.lfp import LFPV1, ImportedLFPV1
 from spyglass.utils.dj_helper_fn import fetch_nwb
+from spyglass.utils.dj_merge_tables import Merge
 
 schema = dj.schema("lfp_merge")
 
 
 @schema
-class LFPOutput(dj.Manual):
+class LFPOutput(Merge):
     definition = """
-    lfp_id: uuid
-    ---
-    source: varchar(40)
-    version: int
-
+    merge_id: uuid
     """
 
     class LFPV1(dj.Part):
         definition = """
         -> master
-        -> LFPV1
         ---
-        -> AnalysisNwbfile
-        lfp_object_id: varchar(40)
+        -> LFPV1
```

Comment:
Hey Chris, I was wondering why you kept `AnalysisNwbfile` in the definition for `ImportedLFPV1` and not `LFPV1` and `CommonLFP`?
```suggestion
        -> LFPV1
        -> AnalysisNwbfile
```

Reply:
Oops, that was an oversight on my part. In each case, I'm interested in removing redundancy and keep the merge table sparse. It's my understanding that each of these tables is connected to `AnalysisNwbFile` immediately upstream, and that connection does not differ from the connection here

Reply:
I'm not totally clear on this, but would the upstream `lfp.LFPV1` table's dependence on `AnalysisNwbfile` be enough to allow `fetch_nwb` to work on these `lfp_merge` tables as currently written?